ignite/engine/__init__.py

```diff
@@ -115,6 +124,16 @@ def supervised_training_step_amp(
     Returns:
         Callable: update function
 
+    Example::
+
```

Comment:
Same here and below

Reply:
Added in https://github.com/pytorch/ignite/pull/1661/commits/8ad516d07c05fa70d33acf93daeca4bac4181d96